public/locales/bn/sidebar.json

```diff
@@ -7,5 +7,14 @@
   "Import data": "আলাপচারিতা ইমপোর্ট",
   "Are you sure?": "আপনি কি নিশ্চিত?",
   "Clear conversations": "কথোপকথন পরিষ্কার করুন",
-  "Export data": "আলাপচারিতা এক্সপোর্ট"
-}
+  "Export data": "আলাপচারিতা এক্সপোর্ট",
```

Comment:
We should probably mark down what has been machine translated in case we ever need manual review. Maybe not though. I assume the other translations were also machine translations w/o review.